src/integration/blockchain/frankencoin/frankencoin-client.ts

```diff
@@ -80,38 +84,41 @@ export class FrankencoinClient {
       }
     `;
 
-    return request<{ fpss: [FrankencoinFpsGraphDto] }>(Config.blockchain.frankencoin.zchfGraphUrl, document).then(
-      (r) => r.fpss,
+    return request<{ fPS: FrankencoinFpsGraphDto }>(Config.blockchain.frankencoin.zchfGraphUrl, document).then(

```

Comment:
Why `fPs` instead of `fps`?

Reply:
Because it's exactly so defined in the Frankencoin DApp,
referring to "https://github.com/Frankencoin-ZCHF/frankencoin-dapp/blob/main/hooks/useFPSQuery.ts"